packages/secrets/package.json

```diff
@@ -46,5 +46,8 @@
     "rebuild": "npx --yes --package=@napi-rs/cli@2.16.2 -- napi build --config src/keyring/napi.json --cargo-cwd src/keyring --platform --release --js=false src/keyring",
     "test": "ava",
     "version": "napi version"
+  },
+  "dependencies": {
+    "@zowe/secrets-for-zowe-sdk": "file:"
```

Comment:
Not sure that adding these is really necessary. 🤔 

Could we remove these? (from all package.json files?)